.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,48 @@
+name: Report a Bug
+description: Report a bug here on our issue tracker.
+body:
+- type: "input"
+  id: "labels"
+  attributes:
+    label: "Labels"
+    description: "To automatically label this issue, add the uppercase'd label(s) sourrounded by brackets"
+    value: "[BUG]"
+    placeholder: "[LABEL]"
+
+- type: "input"
+  id: "description"
+  attributes:
+    label: "Description"
+    description: "Give a short and concise description of the bug"
```

```suggestion
    description: "Give a short description of the bug"
```
Words mean same thing, simpler for ESL

---

Was going off the current issue template